### DISCUSSION TITLE
【不具合】質問の回答への返信コメントでCodePenの埋め込みがうまく表示されない

### DISCUSSION THREAD

**Discussion by Yotty0404** [increments/qiita-discussions#612] [要望・アイデア・不具合]
タイトルの通り、質問内の回答への返信コメントでCodePenの埋め込みが正しく表示されません。
プレビュー時では埋め込みがうまくいっているように見えますが、投稿後はうまく表示されないです。

回答として投稿する場合はCodePenの埋め込みは正しく表示されます。

プレビュー時：
![image](https://github.com/increments/qiita-discussions/assets/74546201/a1bfeef0-b4e1-4b48-b5e5-5d65b05f116e)

投稿後：
![image](https://github.com/increments/qiita-discussions/assets/74546201/f1d9e5c9-ae01-429b-97db-b3d3695ee1d4)

**Comment by tanaka-asami415** [comment]
@Yotty0404 
ご不便をおかけして申し訳ありません🙏
セキュリティの観点から回答に対するコメントでは CodePen が使用できないようにしておりましたが、ゆくゆくは回答コメントでも CodePen が使えるようにしたいと考えています。
直近の開発優先度などから先になりそうなため、一旦プレビューとの違いを修正するかは検討したいと思います🙇

**Comment by wataru86** [comment]
@Yotty0404

お待たせしました！
検討の結果、問題ないと判断し、回答に対するコメントでも CodePen を使えるようにしました！
https://qiita.com/release-notes/2024-05-27
